app/assets/stylesheets/application.css.scss

```diff
@@ -48,6 +48,19 @@
   position: relative;
 }
 
+.admin-photo-container {
+  background-image: url('https://dancer.com/wp-content/uploads/2013/10/Cross-Training.jpg');
+  background-position: center center;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  -webkit-background-size: cover;
+  background-size: cover;
+  height: 100vh;
```

Comment:
Properties should be ordered background-image, background-position, -moz-background-size, -o-background-size, -webkit-background-size, background-size, border-color, border-top, height, position